qtpy/QtLocation.py

```diff
@@ -8,18 +8,27 @@
 """Provides QtLocation classes and functions."""
 
 from . import (
+    parse,
```

Comment:
This only works "by accident"/as an internal implementation detail, as `parse` is in fact imported from `packaging`, not QtPy; thefore, it should be imported from there instead.

Also, AFAIK our existing tests use the cruder `if PYQT_VERSION.startswith("6.5")`, but since `packaging` is required anyway, we may as well use it to be cleaner and more precise, as you do.

Reply:
> This only works "by accident"/as an internal implementation detail, as parse is in fact imported from packaging, not QtPy; thefore, it should be imported from there instead.

I actually did originally have `from packaging import parse` but I for some reason I can't remember, I removed it and did this instead. 

> Also, AFAIK our existing tests use the cruder if PYQT_VERSION.startswith("6.5"), but since packaging is required anyway, we may as well use it to be cleaner and more precise, as you do.

I did see that when I was poking around. I can fix all the `startswith`s.

Reply:
> I did see that when I was poking around. I can fix all the `startswith`s.

Yeah, thanks—that's probably worth doing in a separate PR from this one, just to be clear.